cmd/porter/bundle.go

```diff
@@ -77,6 +79,35 @@ func buildBuildCommand(p *porter.Porter) *cobra.Command {
 	return cmd
 }
 
+func buildBundleListCommand(p *porter.Porter) *cobra.Command {
+	opts := struct {
+		rawFormat string
+		format    printer.Format
+	}{}
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "list bundles",
+		Long:  `List all bundles managed by Porter`,
```

Comment:
We can get a bit more verbose here. Maybe let them know that they can see bundles that have been installed? They may not immediately connect the word "managed" with something that they did with "porter install", you know?

Reply:
Updated; let me know what you think... meanwhile, new ticket to address dissonance between bundles and clams in: https://github.com/deislabs/porter/issues/330